one_off_scripts/__init__.py

```diff
@@ -1,6 +0,0 @@
-import os
```

Comment:
I’m 👍🏻 on removing these if they won’t ever be used again, but its probably best to do that in a separate PR.